e2e-tests/gef/cypress/integration/clone-gef-deal.spec.js

```diff
@@ -0,0 +1,282 @@
+import relative from './relativeURL';
+import automaticCover from './pages/automatic-cover';
+import manualInclusion from './pages/manual-inclusion-questionnaire';
+import applicationDetails from './pages/application-details';
+import aboutExporter from './pages/about-exporter';
+import cloneGEFdeal from './pages/clone-deal';
+import nameApplication from './pages/name-application';
+import mandatoryCriteria from './pages/mandatory-criteria';
+import uploadFiles from './pages/upload-files';
+import statusBanner from './pages/application-status-banner';
+import CREDENTIALS from '../fixtures/credentials.json';
+
+const { format } = require('date-fns');
+
+context('Clone GEF (AIN) deal', () => {
+  let AINapplication;
+  let testApplication;
+  before(() => {
+    cy.reinsertMocks();
+    cy.apiLogin(CREDENTIALS.MAKER).then((token) => token).then((token) => {
+      cy.apiFetchAllApplications(token);
+    }).then(({ body }) => {
+      AINapplication = body.items[2]._id;
+      testApplication = body.items[1]._id;
```

Comment:
minor but should we put id in the variable name so it's clearer?

Reply:
sorted